detox/src/client/AsyncWebSocket.js

```diff
@@ -95,6 +95,11 @@ class AsyncWebSocket {
 
     const messageId = message.messageId;
     const inFlight = this.inFlightPromises[messageId] = new InflightRequest(message).withTimeout(options.timeout);
+
+    if (this.hasMultiplePendingActions()) {
+      throw  new DetoxRuntimeError('Attempting to send a new interaction without responding to the previous one. Previous interaction may be missing await');
```

Comment:
I think should be revised, to a more detox-ish jargon (is that a thing?). More in the direction of:
```
Detox has detected multiple interactions taking place simultaneously. Have you forgotten an await in your test code?
```